docs/config.md

```
@@ -336,7 +355,7 @@ Config YAML の値を環境変数で上書き可能。環境変数が設定さ
 | `WEB_AUTH_TOKEN` | `channels.web.auth_token` | `string` |
 | `WEB_ALLOWED_ORIGINS` | `channels.web.allowed_origins` | `string`（カンマ区切り） |
 | `DISCORD_BOT_TOKEN` | `channels.discord.bots.*.token` | `string` | SecretRef `{ source: env, id: DISCORD_BOT_TOKEN }` で解決（§4 参照）。Layer 2 の直接オーバーライドは非対応 |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)